medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+            exceptions = cache_db_con.select(sql_base, params)
+        else:
+            self.api_finish(status=400, error='bad request')
+
+        exceptions = [{'exception_id': row[0],
+                       'indexer': row[1],
+                       'indexer_id': row[2],
+                       'show_name': row[3],
+                       'season': row[4],
+                       'custom': row[5]}
+                      for row in exceptions]
+
+        self.api_finish(data={"exceptions": exceptions, "last_updates": get_last_updates()})
+
+    def put(self, *args, **kwargs):
+        """Update show information.
+
+        :param show_id:
+        :type show_id: str
+        """
+        exception_id = self._parse(kwargs.pop('row_id'))
+
+        data = json_decode(self.request.body)
+
+        if not all([data.get('indexer_id'),
+                    data.get('season'),
+                    data.get('show_name'),
+                    data.get('indexer')]):
+            return self.api_finish(status=400, error="Invalid post body, can't update")
+
+        cache_db_con = db.DBConnection('cache.db')
+        last_changes = cache_db_con.connection.total_changes
+        cache_db_con.action(b'UPDATE scene_exceptions'
+                            b' set indexer = ?'
+                            b', indexer_id = ?'
+                            b', show_name = ?'
+                            b', season = ?'
+                            b', custom = 1'
+                            b' WHERE exception_id = ?',
+                            [data.get('indexer'),
+                             data.get('indexer_id'),
+                             data.get('show_name'),
+                             data.get('season'),
+                             exception_id])
+        if cache_db_con.connection.total_changes - last_changes == 1:
+            return self.api_finish(status=201)
+        return self.api_finish(status=400, error="Could not update.")
+
+    def post(self, *args, **kwargs):
+        """Add a show."""
+        data = json_decode(self.request.body)
+
+        if not all([data.get('indexer_id'),
+                    data.get('season'),
+                    data.get('show_name'),
+                    data.get('indexer')]):
+            return self.api_finish(status=400, error="Invalid post body, can't update")
+
+        cache_db_con = db.DBConnection('cache.db')
+        last_changes = cache_db_con.connection.total_changes
+        cache_db_con.action(b'INSERT INTO scene_exceptions'
+                            b' (indexer, indexer_id, show_name, season, custom) '
+                            b' values (?,?,?,?,1)',
+                            [data.get('indexer'),
+                             data.get('indexer_id'),
+                             data.get('show_name'),
+                             data.get('season')])
+        if cache_db_con.connection.total_changes - last_changes > 0:
+            return self.api_finish(status=204)
```

Comment:
You're creating a resource. You should return it back in the response since new fields could have been populated (auto generated ids, etc). 
Either you return 201 created with http header with the url location for the resource or http 200 with the resource itself in the response body